src/app/components/home/home.component.html

```diff
@@ -335,6 +336,33 @@ <h2>{{ 'SETTINGS.changeLanguage' | translate }}</h2>
 
     </div>
 
+    <!-- TODO -- TRANSLATE !!! -->
+    <div *ngIf="settingToShow == 4" class="shortcutMenu">
+
+      <span>Hold <em class="key modifierKey">Ctrl</em>
+        <ng-container *ngIf="macVersion">
```

Comment:
This doesn't work on the dev version - I assume it works if you build the app?

Reply:
Thank you for checking -- I'll double check this later. What I had expected was that if you change the `macVersion` to `true` (line 103 in `home.component.ts`) you should be able to see this line of text. The boolean is one of the manual changes needed when building the app for Mac (so that you get the native-looking close, maximize, minimize buttons) ✅

Reply:
Oh! I hadn't noticed that - maybe there's a way to detect the OS you're running on? Save changing it when building/testing the app! 💯